test/cli/test/simple/variable_subs_using_variable_file/values.yaml

```diff
@@ -12,11 +12,15 @@ policies:
   # operation is not provided by user
   - name: test-require-image-tag-pass
     values:
-  # operation is empty
+      request.operation: CREATE
+  # operation is empty, by default it will be set to CREATE
   - name: test-require-image-tag-fail
     values:
       request.operation: ""
```

Comment:
Why would we want this behavior? If the user has explicitly specified an empty value for this variable, we should not implicitly be setting it to create?